src/components/form/TextField/TextField.tsx

```diff
@@ -0,0 +1,78 @@
+import { useState } from 'react';
+import { HelpOutline, EmailOutlined } from '@mui/icons-material';
+
+export type TextFieldProps = {
+  className?: string;
+  placeholder?: string;
+  name?: string;
+  type?: string;
+};
+
+function TextField(propsInput: TextFieldProps) {
+  const [inputValue, setInputValue] = useState<string>('');
+  // const [statusValue, setStatusValue] = useState<string>('');
+  // const statusInput = [
+  //   { status: 'error', color: 'red' },
+  //   { status: 'success', color: 'green' },
+  //   { status: 'withValue', className: 'black' },
+  // ];
+  const { className, ...props } = propsInput;
+
```

Comment:
please fix contruction, we want to have fully control with the props
```suggestion
  const { className, placeholder, name, type} = propsInput;
  
```